timesketch/api/v1/resources.py

```diff
@@ -410,6 +410,16 @@ def get(self, sketch_id):
             for t in sketch.active_timelines
         ]
 
+        # Get event count and size on disk for each index in the sketch.
+        stats_per_index = {}
+        es_stats = self.datastore.client.indices.stats(
+            index=sketch_indices, metric='docs, store')
+        for index_name, stats in es_stats['indices'].items():
+            stats_per_index[index_name] = {
+                'count': stats['total']['docs']['count'],
```

Comment:
also here... do we know that these will always exist?

```
stats.get('total', {}).get('docs', {}).get('count', 0)
```
or something like that

Reply:
Done